src/components/makeResponsive.js

```diff
@@ -72,9 +73,9 @@ function makeResponsive(Wrapped: React.ComponentType<any>): React.ComponentType<
          * @param {Component} react component
          * @return {void}
          */
-        onResize = ({ bounds: { width } }: { bounds: ClientRect }) => {
+        onResize = debounce(({ bounds: { width } }: { bounds: ClientRect }) => {
             this.setState({ size: this.getSize(width) });
-        };
+        }, 500);
```

Comment:
Is this low enough for the change in width to not be noticeable when the window is resized continually (back-and-forth, for example)?

Reply:
It's a bit noticeable, we can adjust later if someone complains, but the use case of window resizing continuously is more of a corner case.